projects/step-core/src/lib/modules/table/shared/table-remote-data-source.ts

```diff
@@ -67,16 +67,17 @@ const convertTableRequest = (req: TableRequest): TableRequestData => {
       : { name };
   });
 
-  const orderColumnIndex = !!req.orderBy ? req.columns.indexOf(req.orderBy.column) : -1;
-  if (orderColumnIndex >= 0) {
-    result.columns[orderColumnIndex].orderable = true;
-    result.order = [
-      {
-        column: orderColumnIndex,
-        dir: req.orderBy!.order,
-      },
-    ];
+  let orderColumnIndex = 0; //if not specified otherwise, the first column is sorted ascendingly
+  if (!!req.orderBy && req.columns.indexOf(req.orderBy.column) >= 0) {
+    orderColumnIndex = req.columns.indexOf(req.orderBy.column);
   }
+  result.columns[orderColumnIndex].orderable = true;
+  result.order = [
+    {
+      column: orderColumnIndex,
+      dir: !!req.orderBy?.order ? req.orderBy.order : 'asc',
+    },
+  ];
```

Comment:
This change looks like we could also replace  `: -1;`  with `: 0;`. I added a discussion point to Trello.